docs/area-owners.md

```diff
@@ -0,0 +1,29 @@
+# Pull Requests Tagging
+
+If you need to tag folks on an issue or PR, you will generally want to tag the owners (not the lead) for [area](#areas) to which the change or issue is closest to.
+
+## Areas
+
+| Area                      | Lead                 | Owners                            | Notes                                                                                             |
+|---------------------------|----------------------|-----------------------------------|---------------------------------------------------------------------------------------------------|
+| area-acquisition          | @joperezr            | @dotnet-area-acquisition          | Issues/PRs related to how Aspire is installed. (Workload or VS Install)                           |
+| area-app-model            | @adityamandaleeka    | @dotnet-area-app-model            | Issues/PRs related to the Aspire app model and Aspire AppHost                                     |
+| area-app-testing          | @ReubenBond          | @dotnet-area-app-testing          | Issues/PRs related to the framework we provide for scenario testing of Aspire apps                |
+| area-components           | @sebastienros        | @dotnet-area-components           | Issues/PRs related to Aspire Components                                                           |
+| area-dashboard            | @kvenkatrajan        | @dotnet-area-dashboard            | Issues/PRs related to the Dashboard UI                                                            |
+| area-deployment           | @davidfowl           | @dotnet-area-deployment           | Issues/PRs related to deploying Aspire apps                                                       |
+| area-docs                 | @IEvangelist         | @dotnet-area-docs                 | Issues/PRs related to documentation                                                               |
+| area-engineering-systems  | @joperezr            | @dotnet-area-engineering-systems  | Issues/PRs related to how the repository is built (build infrastructure)                          |
```

Comment:
what about our test infrastructure

Reply:
@radical do we normally put that in engineering systems? I'd imagine yes, if so I can edit this